depression-and-anxiety/managing-depression-and-anxiety.md

```diff
@@ -60,60 +57,44 @@ Just being active in any way for 20 minutes a day can help lift your mood.
 </div>
  
 ##Keeping in touch with people
-Socialising can help you keeping active and grounded.
+Socialising helps you keep active and grounded.
 
-Keeping in touch with people can feel difficult when you feel depressed or anxious but facing your fears is an important part of managing your condition on a daily basis. 
+Keeping in touch with people can feel difficult when you feel depressed or anxious but facing your fears is an important part of managing your condition every day. 
 
 Try to talk to friends and family regularly and meet up with them whenever possible. 
 
 [Volunteering](https://www.gov.uk/government/get-involved/take-part/volunteer) can also be a good way to meet people while helping others. 
 
-##Stop and notice things
-When we’re busy and stressed we often develop a ‘tunnel vision’ and rush through life. But the way we think is very important to the way we feel. 
+##Noticing the things around you
+The way we think affects the way we feel. When we’re busy and stressed we often develop a ‘tunnel vision’ and rush through life. 
 
-Every day just take a moment to stand back and observe: 
+Every day try to take a moment to stand back and notice: 
 
 - what you’re thinking
 - how you’re feeling 
 - what’s happening around you
 
-Just notice these things without doing anything further. 
+Try not to think of anything else. 
 
 This is called 'mindfulness'. It can help you notice signs of stress, anxiety or depression earlier and deal with them better.
   
-Things like yoga and meditation can also help you being more aware. 
- 
-##Eating
- 
-When your body is healthy, you’ll feel physically better and also feel better about yourself. 
- 
-Try to eat regular meals and drink lots of fluids (at least a litre a day). Dehydration can affect your mood
- 
-### Eat fruit and vegetables every day
- 
-Especially ones rich in vitamins:
+Things like yoga, breathing exercises and meditation can also help you being more aware. 
  
-- apricots, melon, peaches
-- broccoli, carrots, cabbage, pumpkin, spinach, sweet potato
-- nuts and seeds
+##What you eat can affect how you feel
+ When your body is healthy, it helps you feel better about yourself. 
  
-### Eat some starchy foods
+Try not to skip meals drink lots of fluids (preferably water). Dehydration can affect your mood.
  
-Things like:
+Every day try to eat:
  
-- rice, pasta
-- bread
+- fruit and vegetables 
+- starchy foods, like pasta, rice or bread 
+- protein foods, like lean meat, fish, beans, cheese, nuts or yogurt
+
+##Alcohol can make depression worse
+Try not to drink too much alcohol. 
+
+- Men should not have more than 3 to 4 units a day 
+- Women should drink no more than 2 to 3 units a day
```

Comment:
The phrasing of these two lines is different: "should not have more" vs "should drink no more". Is that intentional?